client/components/angular-ui-router/angular-ui-router.module.js

```diff
@@ -1,16 +0,0 @@
-angular
-  .module('managerApp')
-  .config(($urlServiceProvider, $locationProvider) => {
-    $urlServiceProvider.rules.otherwise('/');
-    $locationProvider.html5Mode(false);
-  })
-  .run(($transitions, $state) => {
-    $transitions.onError({}, (transition) => {
-      const error = _.get(transition.error(), 'detail');
-      // manage the specific LOADING_STATE_ERROR code
-      // specific code that handle errors of state resolve
-      if (_.get(error, 'code') === 'LOADING_STATE_ERROR') {
-        $state.go('error', { error });
-      }
-    });
-  });
```

Comment:
I don't think this should be deleted because it doesn't impact the same thing

Reply:
The only case where it is used, is in vps section on this line : https://github.com/ovh-ux/ovh-manager-cloud/blob/master/client/app/vps/vps.js#L35. So I feel confident to delete this :)